Client/src/Pages/FindSupporter/MatchingHome/Matching.js

```diff
@@ -226,38 +228,58 @@ const ResponsiveDrawer = (props) => {
         <AppBar position="fixed" className={classes.appBar}>
           <Menu />
         </AppBar>
-        <Drawer
-          className={classes.drawer}
-          variant="permanent"
-          classes={{
-            paper: classes.drawerPaper,
-          }}
-        > 
-          {/*All of the filters*/}
-          <div>
-          <br/>
-          <br/>
-          <br/>
-          <br/>
-          <br/>
-          <br/>
-          <Typography align="center" variant="h5">Filters</Typography>
-              <br />
+        { isMobile && 
+          <AppBar position="fixed" className={classes.filtersBar}>
+            <Toolbar onClick={e => setFiltersOpen(!filtersOpen)}>
+              <IconButton color="primary">
+                <MenuIcon/>
+              </IconButton>
+              <Typography color="primary">
+                {filtersOpen ? "Hide Filters" : "Show Filters"}
+              </Typography>
+            </Toolbar>
+          </AppBar>
+        }
+        { filtersOpen &&
```

Comment:
(Not related to being mobile responsive)

Why the inconsistency in the same file?

`{ ` on line 231 and `{` on line 259 (notice the extra space). Sorry if I am a bit too extreme but I feel like if we have a proper standard for writing code, it makes it look easy on the eye and is easier for future developers to work on when they are learning the code written by us.

This is something not related to this file in particular but something we could focus on throughout our project.

Reply:
Yeah we should try to all use a vscode extension that auto formats the code. I can look into what a good one would be and let you guys know